docs/sessions.md

```diff
@@ -76,44 +88,46 @@ Cookie Serialization<br>_(of particular interest to .NET Core < netstandard2.0)_
 Suave uses the .NET Framework type `BinaryFormatter` to serialize the `Map<string, obj>` containing the session state; this is the default. However, the `BinaryFormatter` was removed in the .NET Core API, and the `DataContractJsonSerializer` does not recognize the `Map<string, obj>` type. One option is to utilize JSON.NET to serialize this object. To use that, ensure you've added the `Newtonsoft.Json` NuGet package to your project, then put the following code somewhere before the `suaveCfg` definition in the example above.
 
 {% highlight fsharp %}
-/// alias
-let utf8 = System.Text.Encoding.UTF8
-
+// NOTE: UTF8 is in the auto-opened module YoLo.
 type JsonNetCookieSerialiser () =
   interface CookieSerialiser with
     member x.serialise m =
-      utf8.GetBytes (JsonConvert.SerializeObject m)
+      UTF8.bytes (JsonConvert.SerializeObject m)
     member x.deserialise m =
-      JsonConvert.DeserializeObject<Map<string, obj>> (utf8.GetString m)
+      JsonConvert.DeserializeObject<Map<string, obj>> (UTF8.toString m)
 {% endhighlight %}
 
 Then, modify the configuration to use that serializer.
 
 {% highlight fsharp %}
-/// Again, do not make this your server key in production...
 let suaveCfg =
   { defaultConfig with
-      serverKey = System.Text.Encoding.UTF8.GetBytes("12345678901234567890123456789012")
+      serverKey = UTF8.bytes "[the-generated-key]"
```

Comment:
When using the built-in function, there's no need to represent the bytes as chars, thus no need to convert from utf8.